### PR TITLE
dbeaver/pro#9438 Fix: diagrams are exported with black fill between links dark theme

### PR DESCRIPTION
**Pull request by SychevAndrey**

### CODE REVIEW

File: webapp/packages/core-utils/src/svgToBlob.test.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+import { describe, expect, it } from 'vitest';
+
+import { svgToBlob } from './svgToBlob.js';
+
+describe('svgToBlob', () => {
+  it('wraps an SVG string in a Blob with the correct MIME type and XML declaration', async () => {
```
Comment (sergeyteleshev) [inline webapp/packages/core-utils/src/svgToBlob.test.ts:13]: i think we can add edge cases handling here:
1. not an svg passed
2. empty string passed

File: webapp/packages/core-utils/src/svgToDataUri.test.ts [hunk 2]
```diff
@@ -1,25 +0,0 @@
-/*
- * CloudBeaver - Cloud Database Manager
- * Copyright (C) 2020-2025 DBeaver Corp and others
- *
- * Licensed under the Apache License, Version 2.0.
- * you may not use this file except in compliance with the License.
- */
-import { describe, expect, it, vi } from 'vitest';
-import * as utf8ToBase64 from './utf8ToBase64.js';
```
Comment (sergeyteleshev) [inline webapp/packages/core-utils/src/svgToDataUri.test.ts:9]: this also can be deleted since it has no more usages

**Review by sergeyteleshev — APPROVED** [review]
